src/test/java/com/linecorp/armeria/client/http/HttpClientIntegrationTest.java

```diff
@@ -140,6 +148,48 @@ public static void destroy() throws Exception {
         });
     }
 
+    /**
+     * Tests if the {@link HttpClient} can send a minimum possible request.
```

Comment:
Got it after reading a bit more :)

testNoContentRequestHasNoHeaders

I'm OK with encoding this in either the method name or comment. But it's important to make clear why this test exists.